articles/fleet-usage-statistics.md

```diff
@@ -17,6 +17,7 @@ Below is the JSON payload that is sent to Fleet Device Management Inc:
   "numHostsEnrolled": 999,
   "numUsers": 999,
   "numTeams": 999,
+  "numQueries": 999,
```

Comment:
@rachaelshaw, should this be `numSavedQueries` so it's clear these are saved queries?